src/detailPage/DetailCard.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  StyledPlantImage,
```

Comment:
Ich habe heute auch styled components ausgelagert, und die ausgelagerten components nur als PlantImage, Textbox usw. importiert, falls du es doch noch mal in deiner DetailCards extra stylen willst, kannst du dann noch den Zusatz Styled... wählen, sonst müsstest du schreiben StyledStyled das ist vielleicht umständlich